src/main.rs

```diff
@@ -933,6 +943,7 @@ fn rtt_and_heap_info_from(
         };
 
         match name {
+            "main" => main = Some(symbol.address() as u32 & !THUMB_BIT),
```

Comment:
I don't think this symbol is guaranteed to exist in the final binary so we may want to do something else

Reply:
At least when using cortex-m-rt it will always exist, I think